src/main/java/sirius/biz/importer/OnLoadEvent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.importer;
+
+import sirius.db.mixing.BaseEntity;
+import sirius.db.mixing.Entity;
+import sirius.db.mixing.Mapping;
+import sirius.kernel.commons.Context;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Triggered within {@link sirius.biz.importer.ImportHandler#load(Context, BaseEntity)} in order to update an entity
+ * using the given context.
+ *
+ * @param <E> the type of entity being updated
+ */
+public class OnLoadEvent<E extends Entity> extends ContextScriptableEvent<E> {
```

Comment:
I would name it BeforeLoadEvent, just in case we ever need an AfterLoadEvent later on...

Reply:
But its currently invoked after the mappings are loaded

Reply:
Yes. Just the opposite of what I said: rename to AfterLoadEvent so we can have a BeforeLoadEvent.
Besides, as to the current known use-cases we need to change the context BEFORE the load in most of the cases.

Reply:
Naming has been unified and split into BeforeLoad and AfterLoad to support all use-cases (both events receive the entity and the context, so the data can be changed on either side (depending on the evnt type))